### ISSUE TITLE
PDO Error when trying to configure without first setting permissions

### ISSUE THREAD

**Issue by laceysanderson** [tripal/brapi#5]
After installing the modules and loading the MCPD ontology, I attempted to configure the module by going to `admin/tripal/extension/brapi`. On this page I receive the following

```
PDOException: SQLSTATE[42601]: Syntax error: 7 ERROR: syntax error at or near ")" LINE 4: WHERE r.name IN (); ^: SELECT COUNT(DISTINCT(ur.uid)) FROM {users_roles} ur JOIN {role} r ON r.rid = ur.rid WHERE r.name IN ();; Array ( ) in brapi_theme_compute_role_stats() (line 47 of /var/www/dev/tripal/sites/all/modules/brapi/theme/brapi.theme.inc).
```

When navigating to People > Permissions, there were none set for "Breeding API" or "Breeding API site entity".

**Comment by laceysanderson** [comment]
As expected, the Error is resolved by setting permissions :-)

**Comment by guignonv** [comment]
It should be fixed by commit b748d83753b37a936d68cc523fb4bb4ad3078f16
